openedx/core/lib/api/view_utils.py

```diff
@@ -92,7 +92,10 @@ def _decorator(func_or_class):
         Requires either OAuth2 or Session-based authentication.
         If is_user is True, also requires username in URL matches the request user.
         """
+        from edx_rest_framework_extensions.authentication import JwtAuthentication
```

Comment:
well simplest solution I see is to change edx-drf-extensions so that it does not run code on import. 
however, I do not know if other things could depend on this functionality of this module